static/js/compose.js

```diff
@@ -565,8 +565,20 @@ exports.validate = function () {
 };
 
 exports.handle_keydown = function (event) {
-    var code = event.keyCode || event.which;
     var textarea = $("#compose-textarea");
+
+    // Set the rtl class if the text has an rtl direction, remove it otherwise
+    var text = textarea.val();
```

Comment:
It's worth extracting these 4 lines of code to a function called `strip_quoting` or something similar, since I can foresee this logic getting a bit more complex to deal with syntax like `> ` and maybe other things.  It also makes it super easy to write a node test for it.

Reply:
In fact, the only thing I could think of that can make a problem is ```` ```quote ````. Things like `>` don't make a difference, because they don't have any "ltr" characters in them. (">" is a neutral character.)

Reply:
Ok, fair enough, I would probably still extract it for testability/modularity, but it's not a big deal.